front/src/modules/trainschedule/components/Timetable/consts.ts

```diff
@@ -7,8 +7,13 @@ export const specialCodeDictionary: { [key: string]: string } = {
 export const invalidTrainValues: {
```

Comment:
Seems like we don't use that `const` anywhere in the app ? If we don't need it, you can remove it as well with the `InvalidReason` type which is used only here.